### ISSUE TITLE
about baseline code

### ISSUE THREAD

**Issue by Seonghak35** [HychaoWang/SleepKD#1]
Hi, your paper is very interesting and I would like to develop it further, but I don't know how you implemented the baseline code, I would like to make an equivalent comparison in the same situation, can you tell me what code you implemented to get these results or did you write the whole code from scratch? The code you published now only introduces loss.py.

**Comment by HychaoWang** [comment]
In keras, it is a little tricky to implement kd with intermediate features. The loss.py we published can be treated as a layer for kd. You should add the layer into your model and set the inputs the layer needs. In train.py, you need to extract the intermediate features from teacher models as inputs of the student models. We will update the readme recently to help you implement.

**Comment by HychaoWang** [comment]
We did implement baselines from scratch, because none of them is implemented in keras by their authors.
